### ISSUE TITLE
Integrate the itkTBBTestFilter class inside itkTBBImageToImageFilterTest.cxx

### ISSUE THREAD

**Issue by jhlegarreta** [rdebroiz/ITKTBBImageToImageFilter#4]
If a helper class is required to run the test, this is usually declared/defined inside the test file itself.

You may fin other examples in ITK, such as:
```
Modules\Core\Common\test\itkColorTableTest.cxx
Modules\Segmentation\BioCell\test\itkBioCellTest.cxx
```
You may look for them typing _Helper_ for `.cxx` files; other helper methods are named _run{WhateverTheTestNameIs}_.

**Comment by jhlegarreta** [comment]
Is the header file left behind necessary?

**Comment by rdebroiz** [comment]
Of course not, sorry i forgot to `git rm` the files.

**Comment by jhlegarreta** [comment]
Thanks. Addressed in commits d3e0a4c058cb0426ae253909b816cff30224d7ee and 276000fe6095f55cf8e3a73754fc40c7883f4e27.
